### DISCUSSION TITLE
v2.0.0预计何时正式发布？

### DISCUSSION THREAD

**Discussion by jilieryuyi** [milvus-io/milvus#7068] [Q&A and General discussion] [answered]
如题

**Comment by yhmo** [comment, marked as answer]
Due to some technical issues, the RC3 is delay for over one week. We decide to skip RC3, and directly publish RC4. Now the release work of RC4 is almost done. The RC4 docker image has been uploaded to docker hub, and we will release the documentation/sdk later.
